### DISCUSSION TITLE
How to Include "中国科学技术大学学位论文原创性声明" page in overleaf version of USTC thesis

### DISCUSSION THREAD

**Discussion by MOmarFarooq** [ustctug/ustcthesis#331] [Q&A]
Please guide me to include the "中国科学技术大学学位论文原创性声明" page in overleaf version of USTC thesis. By default it is not showing there.  Also "List of Tables" and "List of Figures" are not showing after the "Contents".

Kindly response me on that.

**Comment by zepinglee** [comment]
> Please guide me to include the "中国科学技术大学学位论文原创性声明" page in overleaf version of USTC thesis. By default it is not showing there.

This part is generated with `\copyrightpage` command which already exists in the default example.

https://github.com/ustctug/ustcthesis/blob/6e7abb15cd183f4097fee9732f871af358423406/main.tex#L31

> Also "List of Tables" and "List of Figures" are not showing after the "Contents".

It's easy to uncomment the commands.

https://github.com/ustctug/ustcthesis/blob/6e7abb15cd183f4097fee9732f871af358423406/main.tex#L35-L37

Are you using <https://www.overleaf.com/latex/templates/latex-template-for-ustc-thesis/wcwjfrxqbgxp>?
